cdap-ui/app/cdap/components/EntityListView/index.js

```diff
@@ -410,7 +423,57 @@ class EntityListView extends Component {
     });
   }
 
+  dismissSplash() {
```

Comment:
Right, but in an insecure environment (where there is no auth token), then its true that one user dismissing the splash will make it not appear for other future users?